_dprhtml/features/other-dialogs/init.js

```diff
@@ -93,12 +93,54 @@ class OtherDialogsViewModel {
   }
 
   showBookmarksDialog() {
-    this.bookmarkInput('');
     $('#bookmark-dialog-root').modal('show');
   }
 
-  sendBookmarkFromDialog(loc, name, desc) {
-    // TODO when bookmarks is implemented
+  sendBookmarkFromDialog() {
+
+    var loc = __navigationTabViewModel.placeArray();
+    var name = $('#dialog-bookmarkInput').val();
```

Comment:
there is ko binding already on dialog-bookmarkInput - lets use that and not jquery

Reply:
done